subsys/bluetooth/services/hids.c

```diff
@@ -392,6 +392,12 @@ static ssize_t hids_feat_rep_write(struct bt_conn *conn,
 				   void const *buf, u16_t len, u16_t offset,
 				   u8_t flags)
 {
+	/* Write command operation is not allowed for this characteristic. */
+	if ((flags & BT_GATT_WRITE_FLAG_CMD) == BT_GATT_WRITE_FLAG_CMD) {
+		LOG_DBG("Feature Report write command received. Ignore received data.");
+		return 0;
```

Comment:
We may consider returning an error here (just to stick with Zephyr function documentation - for write without response we don't send ATT error response).

Reply:
That is way I returned 0 here. BLE spec said that write without response should be ignored when it is not supported. The 0 value here means that 0 bytes was written. Alternatively, Do you have a suggestion for which ATT error code is suitable ?

Reply:
To be honest I'm not sure. Maybe we could use BT_ATT_ERR_NOT_SUPPORTED. What do you think @joerchan?
Or simply leave zero here, since we do not sent the error code?

Reply:
I think BT_ATT_ERR_NOT_SUPPORTED would be better here just for readability, but other than that they are the same, since the error code will not be used.

Reply:
Ok done. BT_ATT_ERR_NOT_SUPPORTED  is returned.